test/testcafe/spec/DeviceEnrollmentTerminalView_spec.js

```diff
@@ -139,4 +139,145 @@ test
+
+    // go back then switch to the next page when OV account is setup
+    await deviceEnrollmentTerminalPage.clickBackLink();
+    await t.expect(deviceEnrollmentTerminalPage.getBackLink().exists).eql(false);
+    await deviceEnrollmentTerminalPage.clickWithOVAccount();
+    await deviceEnrollmentTerminalPage.clickNextButton();
+    await t.expect(deviceEnrollmentTerminalPage.getBackLink().exists).eql(true);
+
+    await deviceEnrollmentTerminalPage.clickBackLink();
+    await t.expect(deviceEnrollmentTerminalPage.getBackLink().exists).eql(false);
+    await t.expect(deviceEnrollmentTerminalPage.getHeader()).eql('Additional setup required to use Okta FastPass');
+  });
+
+test
+  .requestHooks(logger)('shows the correct content in Android ODA terminal view when Okta Verify is not installed in App Link flow', async t => {
+    const deviceEnrollmentTerminalPage = await setup(t);
+    await rerenderWidget({
+      'proxyIdxResponse': {
+        'deviceEnrollment': {
+          'type': 'object',
+          'value': {
+            'name': 'oda',
+            'platform': 'ANDROID',
+            'enrollmentLink': '',
+            'vendor': '',
+            'signInUrl': 'https://rain.okta1.com',
+            'challengeMethod': 'APP_LINK',
+            'orgName': 'testOrg'
+          }
+        }
+      }
+    });
+    // switch to the next page when OV account is not setup
+    await deviceEnrollmentTerminalPage.clickWithoutOVAccount();
+    await deviceEnrollmentTerminalPage.clickNextButton();
+    await t.expect(deviceEnrollmentTerminalPage.getHeader()).eql('Set up an Okta Verify account');
+    await t.expect(deviceEnrollmentTerminalPage.getSubHeader()).eql('To sign in with Okta FastPass, you’ll need to set up Okta Verify on this device.');
+    await t.expect(deviceEnrollmentTerminalPage.getContentByIndex(1)).contains('If you don’t have Okta Verify installed,');
+    await t.expect(deviceEnrollmentTerminalPage.getContentByIndex(1)).contains('download the app.');
+    await t.expect(deviceEnrollmentTerminalPage.getContentByIndex(2)).eql('Open Okta Verify and follow the steps to add your account.');
+    await t.expect(deviceEnrollmentTerminalPage.getContentByIndex(3)).contains('When prompted, choose Sign In, then enter the sign-in URL:');
+    await t.expect(deviceEnrollmentTerminalPage.getContentByIndex(3)).contains('https://rain.okta1.com');
+    await t.expect(deviceEnrollmentTerminalPage.getCopyOrgLinkButtonLabel()).eql('Copy sign-in URL to clipboard');
+    await t.expect(deviceEnrollmentTerminalPage.getContentByIndex(4)).eql('Finish setting up your account in Okta Verify, then try signing in again.');
+    await t.expect(deviceEnrollmentTerminalPage.getBackLink().exists).eql(true);
+    await t.expect(deviceEnrollmentTerminalPage.getBackLinkText()).eql('Back');
+  });
+
+test
+  .requestHooks(logger)('shows the correct content in Android ODA terminal view when Okta Verify is installed in App Link flow', async t => {
```

Comment:
nit: it might be better to describe the test based on if an account is setup in OV instead of `Okta Verify is installed`. Same suggestion for above test.